src/main.py

```diff
@@ -68,13 +70,21 @@ def main(geo_ls=["zip", "county"]):
     and save jsons to file
     '''
     # from src import dict_merge
+
+    print("Reading WIC Data")
+    wic_start_time = time.time()
+    src.wic.read_wic_data()
```

Comment:
Do we want to start passing options to the main script to opt in or out of reading specific data sources?

Reply:
For this specific data source, lets check if the output file already exists and avoid running it again if it does.